--- android.py.old	2018-06-28 14:40:30.707576902 -0300
+++ android.py	2018-06-28 16:48:04.507770072 -0300
@@ -786,21 +786,16 @@
             # maybe the hook fail because the apk is not
             pass
 
-        # XXX found how the apk name is really built from the title
-        gradle_files = ["build.gradle", "gradle", "gradlew"]
-        is_gradle_build = any((
-            exists(join(dist_dir, x)) for x in gradle_files))
-        if is_gradle_build:
-            # on gradle build, the apk use the package name, and have no version
+        # copy to our place
+	try:
             packagename = config.get('app', 'package.name')
             apk = u'{packagename}-{mode}.apk'.format(
                 packagename=packagename, mode=mode)
             apk_dir = join(dist_dir, "build", "outputs", "apk")
             apk_dest = u'{packagename}-{version}-{mode}.apk'.format(
                 packagename=packagename, mode=mode, version=version)
-
-        else:
-            # on ant, the apk use the title, and have version
+            copyfile(join(apk_dir, apk), join(self.buildozer.bin_dir, apk_dest))
+	except:
             bl = u'\'" ,'
             apptitle = config.get('app', 'title')
             if hasattr(apptitle, 'decode'):
@@ -812,9 +807,7 @@
                 mode=mode)
             apk_dir = join(dist_dir, "bin")
             apk_dest = apk
-
-        # copy to our place
-        copyfile(join(apk_dir, apk), join(self.buildozer.bin_dir, apk_dest))
+            copyfile(join(apk_dir, apk), join(self.buildozer.bin_dir, apk_dest))
 
         self.buildozer.info('Android packaging done!')
         self.buildozer.info(
